include/openthread/thread.h

```diff
@@ -86,11 +86,14 @@ typedef struct otLinkModeConfig
 /**
  * This structure holds diagnostic information for a neighboring Thread node
  *
+ * `mUptime` requires `OPENTHREAD_CONFIG_UPTIME_ENABLE`.
+ *
  */
 typedef struct
 {
     otExtAddress mExtAddress;           ///< IEEE 802.15.4 Extended Address
-    uint32_t     mAge;                  ///< Time last heard
+    uint32_t     mAge;                  ///< Seconds since last heard
+    uint32_t     mUptime;               ///< Seconds since link establishment
```

Comment:
Rather than `uptime`, maybe refer to it as `ConnectionTime`?

Reply:
Thanks. Will rename in next push.